lib/twingly/url.rb

```diff
@@ -6,7 +6,12 @@
 require_relative "url/error"
 require_relative "version"
 
-PublicSuffix::List.private_domains = false
+PublicSuffix::List.default = PublicSuffix::List.parse(
+  File.read(PublicSuffix::List::DEFAULT_LIST_PATH), private_domains: false)
+PublicSuffix::List.default.indexes.keys.
+  map { |name| Addressable::IDNA.to_ascii(name) }.
+  select { |name| name =~ /xn\-\-/ }.
```

Comment:
I used the same expressions as https://github.com/weppos/publicsuffix-ruby/blob/v2.0.2/test/psl_test.rb#L37-L38, but that expression might be constructed to handle more variants (https://github.com/weppos/publicsuffix-ruby/blob/v2.0.2/test/tests.txt)

we can check if we get the same count using `^`